packages/relay-kit/src/packs/safe-4337/estimators/PimlicoFeeEstimator.ts

```diff
@@ -19,6 +26,26 @@ export class PimlicoFeeEstimator implements IFeeEstimator {
     }
   }
 
+  async getPaymasterEstimation({
+    userOperation,
+    paymasterUrl,
+    entryPoint,
+    sponsorshipPolicyId
+  }: EstimateSponsoredFeeFunctionProps): Promise<EstimateSponsoredGasData> {
+    const paymasterClient = new ethers.JsonRpcProvider(paymasterUrl, undefined, {
+      batchMaxCount: 1
+    })
+
+    const params = [userOperationToHexValues(userOperation), entryPoint]
+
+    const gasEstimate = await paymasterClient.send(
+      'pm_sponsorUserOperation',
```

Comment:
Perhaps we can add this and the one in the PimlicoEstimator in the constants file same as other RPC calls

Reply:
done!